aptos-move/framework/aptos-experimental/sources/trading/market/market.move

```diff
@@ -478,64 +489,66 @@ module aptos_experimental::market {
         remaining_size: &mut u64,
         is_bid: bool,
         metadata: M,
-        order_id: u64,
+        order_id: OrderIdType,
         callbacks: &MarketClearinghouseCallbacks<M>,
         fill_sizes: &mut vector<u64>
     ): Option<OrderCancellationReason> {
         let result =
-            self.order_book.get_single_match_for_taker(price, *remaining_size, is_bid);
-        let (maker_order, maker_matched_size) = result.destroy_single_order_match();
-        let (maker_address, maker_order_id) =
-            maker_order.get_order_id().destroy_order_id_type();
-        if (!self.config.allow_self_trade && maker_address == user_addr) {
-            self.cancel_maker_order_internal(
-                &maker_order,
-                maker_order_id,
-                maker_address,
-                std::string::utf8(b"Disallowed self trading"),
-                maker_matched_size,
-                callbacks
-            );
-            return option::none();
-        };
-        let fill_id = self.next_fill_id();
-        let settle_result =
-            callbacks.settle_trade(
-                user_addr,
-                maker_address,
-                order_id,
-                maker_order_id,
-                fill_id,
-                is_bid,
-                maker_order.get_price(), // Order is always matched at the price of the maker
-                maker_matched_size,
-                metadata,
-                maker_order.get_metadata_from_order()
-            );
+            self.order_book
+                .get_single_match_for_taker(price, *remaining_size, is_bid);let (
```

Comment:
formatting is messed up here, `let` should go in a new line.

Reply:
hmm this is coming from

`aptos move fmt --package-path aptos-move/framework/aptos-experimental`